tests/test_compression.py

```diff
@@ -188,6 +188,19 @@ def test_small_msprime_complex_mutations(self):
                 )
         self.verify(tables.tree_sequence())
 
+    def test_ref_seq(self):
+        ts = msprime.simulate(10, recombination_rate=1, mutation_rate=2, random_seed=2)
+        tables = ts.tables
+        tables.reference_sequence.metadata_schema = (
+            tskit.MetadataSchema.permissive_json()
+        )
+        tables.reference_sequence.metadata = {"some": "data"}
+        tables.reference_sequence.data = "ACTG"
+        # NOTE: it's unclear whether we'll want to have this set at the same time as
+        # 'data', but it's useful to have something in all columns for now.
+        tables.reference_sequence.url = "http://example.com/a_reference"
+        self.verify(tables.tree_sequence())
```

Comment:
I assume ``verify`` calls ``tables.assert_equals()`` somewhere along the line? Otherwise we won't actually be testing much here I think.

Reply:
I have verified that `verify` actually verifies. I changed to use `assert_equals` anyway as it gives better errors.